aws_lambda_fsm/constants.py

```diff
@@ -17,6 +17,9 @@
 ################################################################################
 
 
```

Comment:
`# system imports`

Reply:
Really? It's a single import.

Reply:
yes. stylistic consistency is important to to the "steward" of this open source repo (me)